src/index.tsx

```diff
@@ -4,6 +4,12 @@ import App from './App'
 import { BrowserRouter } from 'react-router-dom'
 import GlobalStyles from 'styles/GlobalStyles'
 
+//import mocks/browser.ts file conditionally
+if (process.env.NODE_ENV === 'development') {
```

Comment:
```suggestion
if (window.Cypress) {
```

I would only use this in your Cypress tests, probably don't want to use MSW for your dev server